### ISSUE TITLE
[IAML/GENRES] Adding minimal hierachy

### ISSUE THREAD

**Issue by pasqLisena** [DOREMUS-ANR/knowledge-base#39]
Hello,
I would like to add the following relations to the vocabulary and create 2 new concepts.
The reason are more or less the same of the ones of the issue #38 .

```turtle
<http://data.doremus.org/vocabulary/iaml/genre/lit> skos:narrower <http://data.doremus.org/vocabulary/iaml/genre/vrl> , <http://data.doremus.org/vocabulary/iaml/genre/lid> , <http://data.doremus.org/vocabulary/iaml/genre/lod>, <http://data.doremus.org/vocabulary/iaml/genre/lyh> .

<http://data.doremus.org/vocabulary/iaml/genre/i> a skos:Concept ;
skos:prefLabel "Instrumental music"@en ;
skos:narrower
<http://data.doremus.org/vocabulary/iaml/genre/tri>, <http://data.doremus.org/vocabulary/iaml/genre/aub> , ...

<http://data.doremus.org/vocabulary/iaml/genre/v> a skos:Concept ;
skos:prefLabel "Vocal music"@en ;
skos:narrower
<http://data.doremus.org/vocabulary/iaml/genre/czp>, <http://data.doremus.org/vocabulary/iaml/genre/due> , ...
```

My points: 
- `lit` has the note "When any more specific code can’t be used"
- `vrl`, `lod`, `lyh` have a skos:note "Liturgical", while `lid` has it in the name
- lots of concepts have the skos:note "Instrumental" or "Vocal"

I have also to say that we have some concept from other vocabularies that we can reuse:
- For instrumental we have already http://data.doremus.org/vocabulary/itema3/genre/84 , http://data.doremus.org/vocabulary/itema3/genre/musdoc/81 and http://data.doremus.org/vocabulary/diabolo/genre/musique_instrumentale
- For vocal music we have http://data.doremus.org/vocabulary/itema3/genre/musdoc/76 and http://data.doremus.org/vocabulary/diabolo/genre/musique_vocale

**Comment by rtroncy** [comment]
I'm not questioning the need for those new ```skos:Concept```. However, I found awkward to define those new terms in the ```<http://data.doremus.org/vocabulary/iaml/genre/>``` concept scheme because there are not (yet) defined in IAML.

Should we try first to ask IAML whether they would be interested to add those concepts too? If not, should we not define those concepts in the reconciled concept scheme (pivot graph) that aligns IAML with other genre vocabularies?

**Comment by pasqLisena** [comment]
> Should we try first to ask IAML whether they would be interested to add those concepts too?

It would be interesting. It needs a discussion in the modelling group.

> If not, should we not define those concepts in the reconciled concept scheme (pivot graph) that aligns IAML with other genre vocabularies?

Yes, but the pivot graph does not exists in this moment :)

**Comment by rtroncy** [comment]
@kgtodorov Can you let us know the status regarding the validation of all alignments between controlled vocabularies, and in particular, for genres and mop? Do we have a chance to have a reconciled graphs before the next general meeting?

**Comment by kgtodorov** [comment]
Luckily, my student in charge of that is back to work after some hard times. We have now all new alignments (mop and genres), which are of significantly higher coverage than the old ones thanks to the new algorithm using directly SKOS (so no conversion to OWL). However, we need to go again through the validation process before we leverage the final alignments, this would definitely take more time than the couple of weeks before the AG.

**Comment by pasqLisena** [comment]
So I postpone it after #22

**Comment by pasqLisena** [comment]
Modeling meeting decisions:
- add it in the pivot vocabulary
- propose to IAML the adoption of this hierarchy (I am writing an email with Francoise)
